rfcs/054-prismic-api/README.md

```diff
@@ -0,0 +1,110 @@
+- The elasticsearch index mapping should represent the contract between the pipeline and the API. The API shouldn't need to know anything about the structure of the data in prismic.
+
+## Proposed endpoints
+
+### `/articles`
+
+Filter for type eg stories, webcomics
+
+#### Different kinds of thing (are these 'labels'?)
+
+- story <https://wellcomecollection.org/articles/Y9vktREAACYA1v2U> at /articles
+- webcomic <https://wellcomecollection.org/articles/Y86rshEAACcAI4g4> at /articles
+- book <https://wellcomecollection.org/books/Y7QWrREAAHC43oH0> at /books
+- series <https://wellcomecollection.org/series/WfpEBCoAACrdd43y> composed of articles
+- in pictures <https://wellcomecollection.org/articles/Y-oAIhAAACAAix32> at /articles
+
+#### Filters
+
+- date
+- contributor
+- series
+- contentType
+
+#### Sort
+
+- date
+- relevance
+
+### `/exhibitions`
+
+#### Filter
+
+- date
+- status
+- location
+- contributor?
+- format (ie permanent, exhibition, installation)
+- season
+
+#### Sort
+
+- date
+- relevance
+
+### `/events`
+
+#### Filter
+
+- date
+- accessibility status (is it audio described, signed, etc)
+- location
+- format (eg discussion)
+- event-series
+- season
+- audience (eg schools)
+- contributor
+- isOnline?
+- availableOnline?
+
+#### Sort
+
+- date
+- relevance
+
+### `/books`
+
+?????????
+
+## Open questions
+
+- How do we represent functional content?
+- Should we include the full prismic API response at eg `/articles/{id}`?
+    - should we be able to list all of the docs of a given type at once? what should that url look like?
+- If we want to avoid maintaining multiple versions of the same code, should this work be done as an extension to the concepts API?
+- Works exist at [api.wellcomecollection.org/catalogue/v2/works](api.wellcomecollection.org/catalogue/v2/works), concepts at [/catalogue/v2/concepts](api.wellcomecollection.org/catalogue/v2/concepts), images at [/catalogue/v2/images](api.wellcomecollection.org/catalogue/v2/images).  
+Stories etc aren't part of the catalogue - should the URL be different?
+- what are the top level types? is it articles? stories? series?
+- where do books go? they appear on the stories landing page, so should they appear in stories search results? or are they at a separate endpoint?
+- what is a serial? what is a series?
+- is 'In pictures' the same as 'Image gallery'?
+- Should we be able to filter by contributor role? is that something for the first pass or something we address later? which contributor roles are there? yes for stories? no for exhibitions and events?
+- can we filter exhibitions for accessibility, ie whether a digital guide exists? we can for events, but unclear for exhibitions
+- do we have any exhibition contributors who have contributed to more than one? is a filter for exhibition contributors really going to be useful?
```

Comment:
@harrisonpim to investigate

Reply:
We have results!

| Name                                              |   Count |
|:----------------------------------------------|----:|
| heart-n-soul-at-the-hub                       |   2 |
| ann-veronica-janssens                         |   1 |
| grace-ndiritu                                 |   1 |
| young-trans-people-from-gendered-intelligence |   1 |
| jason-barker                                  |   1 |
| dr-jana-funke                                 |   1 |
| maria-mckinney                                |   1 |
| professor-michael-doherty                     |   1 |
| professor-david-machugh                       |   1 |
| john-walter                                   |   1 |
| alison-rodger                                 |   1 |
| marion-coutts                                 |   1 |
| daria-martin                                  |   1 |
| dr-michael-banissy                            |   1 |
| jim-naughten                                  |   1 |
| martina-amati                                 |   1 |
| professor-kevin--fong                         |   1 |
| ranjit-kandalgaonkar                          |   1 |
| photobot.co                                   |   1 |
| heart-n-soul                                  |   1 |

Looks like Heart n Soul have contributed to multiple exhibitions, and given [this comment](https://github.com/wellcomecollection/docs/pull/90#discussion_r1114170705), i think it's worth including the filter!